specs/osquerybeat.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+inputs:
+  - name: osquery
+    description: "Osquery"
+    platforms:
+      - linux/amd64
+      - linux/arm64
+      - linux/ppc64le
```

Comment:
nit: osquery supports amd64 and arm64 only. don't think there is windows arm64 distro available. we don't have windows for arm either to test if the is any compatibility layer that allows to run x64 on arm windows.